From ef5e7a185e2d8e6496c61af77e81abaaafc2b85b Mon Sep 17 00:00:00 2001
From: Marian Chereji <marian.chereji@nxp.com>
Date: Mon, 16 Jan 2017 10:09:07 +0200
Subject: [PATCH 1/8] wireless: Extend WiFi handshake timeouts for simulations

Extended the authentication and association timeouts so that the
UA systems have a fair chance to complete WiFi handshake when
running on a UA simulator. This patch should be discarded when
running on real UA boards.

Signed-off-by: Marian Chereji <marian.chereji@nxp.com>
---
 net/mac80211/mlme.c | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/net/mac80211/mlme.c b/net/mac80211/mlme.c
index 26053bf..af564c1 100644
--- a/net/mac80211/mlme.c
+++ b/net/mac80211/mlme.c
@@ -31,14 +31,25 @@
 #include "rate.h"
 #include "led.h"
 
+/*
 #define IEEE80211_AUTH_TIMEOUT		(HZ / 5)
 #define IEEE80211_AUTH_TIMEOUT_LONG	(HZ / 2)
 #define IEEE80211_AUTH_TIMEOUT_SHORT	(HZ / 10)
+*/
+#define IEEE80211_AUTH_TIMEOUT		(5 * HZ)
+#define IEEE80211_AUTH_TIMEOUT_LONG	(5 * HZ)
+#define IEEE80211_AUTH_TIMEOUT_SHORT	(5 * HZ)
 #define IEEE80211_AUTH_MAX_TRIES	3
+/*
 #define IEEE80211_AUTH_WAIT_ASSOC	(HZ * 5)
 #define IEEE80211_ASSOC_TIMEOUT		(HZ / 5)
 #define IEEE80211_ASSOC_TIMEOUT_LONG	(HZ / 2)
 #define IEEE80211_ASSOC_TIMEOUT_SHORT	(HZ / 10)
+*/
+#define IEEE80211_AUTH_WAIT_ASSOC	(HZ * 20)
+#define IEEE80211_ASSOC_TIMEOUT		(HZ * 5)
+#define IEEE80211_ASSOC_TIMEOUT_LONG	(HZ * 5)
+#define IEEE80211_ASSOC_TIMEOUT_SHORT	(HZ * 5)
 #define IEEE80211_ASSOC_MAX_TRIES	3
 
 static int max_nullfunc_tries = 2;
-- 
2.1.0.27.g96db324

